mindsdb/integrations/handlers/statsforecast_handler/statsforecast_handler.py

```diff
@@ -34,6 +34,7 @@ def get_season_length(frequency):
     season_dict = {  # https://pandas.pydata.org/docs/user_guide/timeseries.html#timeseries-offset-aliases
         "H": 24,
         "M": 12,
+        "MS": 12,
```

Comment:
Good catch, I've ran into this previously. I recall there's also the option for doing "[X]F", with `X` an integer and `F` the frequency. That would definitely be a new feature though.